packages/vite/src/utils/generator-utils.ts

```diff
@@ -559,6 +559,14 @@ export function createOrEditViteConfig(
     ],
     `;
 
+  const workerOption = `
+    // Uncomment this if you are using workers. 
```

Comment:
I don't think the code needs to be commented out initially.

If you are not using worker the config is not used anyway.

Reply:
For now we just want to keep the config small, so users can choose to delete it if they don't need the entry.

We're discussing a future Nx plugin for Vite that can encapsulate this so the tsconfig paths don't need to be manually configured like this.